lnrpc/chainrpc/chainnotifier.swagger.json

```diff
@@ -357,5 +295,43 @@
         }
       }
     }
+  },
+  "x-stream-definitions": {
```

Comment:
it seems like your `grpc-gateway` version is older than the currently required `1.14.3`.